ansible/roles/kaui/tasks/main.yml

```diff
@@ -50,6 +50,17 @@
     - name: logback.xml
   tags: kaui-logback
 
+- name: Recursively change ownership for the expanded ROOT
+  become: true
+  file:
+    path: "{{ catalina_base }}/{{ kaui_webapps }}/ROOT"
+    state: directory
+    recurse: yes
+    owner: "{{ tomcat_owner }}"
+    group: "{{ tomcat_group }}"
+    mode: u=rwX,g=rX,o=rX
```

Comment:
@sbrossie Do you really need `X`?

Reply:
This task allows to combine both: I am not using small `x`, but big `X`: 

```
+X            make a directory or file searchable/executable by everyone if it is already searchable/executable by anyone.
````